test/MathOptInterface/MOI_wrapper.jl

```diff
@@ -37,14 +37,17 @@ function test_runtests()
             "test_linear_Indicator_ON_ZERO",
             "test_linear_Indicator_constant_term",
             "test_linear_Indicator_integration",
+            # exclude tests that trigger bugs in COPT 6.5.x
+            "test_linear_Semiinteger_integration",
+            "test_linear_Semicontinuous_integration",
```

Comment:
Are these bugs in COPT.jl or the upstream solver?

Reply:
these are bugs in the upstream solver, which will be fixed in an upcoming patch release.

Reply:
> these are bugs in the upstream solver

Selfishly, yay! Glad that our tests are useful. I think these ones were added when we found bugs in Xpress.